packages/cli/src/init/generateMetadataJson.ts

```diff
@@ -8,7 +8,9 @@ export default function generateMetadataJson(modes: Array<'Light' | 'Dark' | 'Co
   return {
     tokenSetOrder: [
       'primitives/modes/globals',
-      'primitives/modes/typography/default',
+      'primitives/size/default',
+      'primitives/modes/typography/primary/theme',
+      'primitives/modes/typography/secondary/theme',
```

Comment:
Æ skjønne det itj helt, men det er nok fordi jeg ikke har vært nok borti dette helt enda :)

Reply:
Dette er filer for Token Studio pluginen :)